docs/Configuration.md

```diff
@@ -1430,6 +1430,12 @@ class CustomReporter {
 module.exports = CustomReporter;
 ```
 
+:::note
+
+For the full list of hooks and argument types see the `Reporter` interface in [packages/jest-reporters/src/types.ts](https://github.com/jestjs/jest/blob/main/packages/jest-reporters/src/types.ts).
+
+:::
```

Comment:
Similar to: https://github.com/jestjs/jest/blob/fb7d95c8af6e0d65a8b65348433d8a0ea0725b5b/docs/CodeTransformation.md?plain=1#L135-L139